WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PostRequests/RegisterInputFailureTests.cs

```diff
@@ -286,35 +293,6 @@ public async Task InputImmatureAsync()
 		await arena.StopAsync(CancellationToken.None);
 	}
 
-	[Fact]
```

Comment:
It is not possible to create or verify ownership proofs for non-segwit scripts so, this test is impossible now.